File: session1/task1/crash.py
# Run this program and observe what happens

def assign_values(x):
    for i in range(100):
        x[i] = (i + 1) * (i + 1);

if __name__ == "__main__":
    data = [0] * 10
    assign_values(data)
    print("Done!")

File: session1/task1/crash.c
/* Compile & run this program and observe what happens */

#include <stdio.h>

void assign_values(int x[])
{
    for (int i = 0; i < 100; ++i) {
        x[i] = (i + 1) * (i + 1);
    }
}

int main(void)
{
    int data[10] = { 0 };

    assign_values(data);
    printf("Done!\n");

    return 0;
}
